### PR TITLE
Feature flask update profile

### PR DESCRIPTION
**Pull request by denicahh**
Add code back into app.py so the html profile page can display the user profile details.

### CODE REVIEW

**Comment by SDzartov** [comment]
Please review, you have stashed changes in your commit, please clean it up to the final version of changes you want to submit.

**Comment by SDzartov** [comment]
@denicahh I assume that his has been done with help on the other branch?

File: neo_dolfin/app.py
Comment (Aditya-Rajgor) [inline neo_dolfin/app.py:1]: Please update the `app.py`, it seems to have comments from the git `rebase`.

File: neo_dolfin/templates/profile.html
Comment (Aditya-Rajgor) [inline neo_dolfin/templates/profile.html:1]: Same for profile.html file.

**Comment by Aditya-Rajgor** [comment]
The files `app.py` and `profile.html` have git comments and code. Please resolve the conflicts and push again

**Pull request closed without merging**